.github/CHANGELOG.md

```diff
@@ -18,6 +20,8 @@
 
 This release contains contributions from (in alphabetical order):
 
+Antonín Hoskovec
+
 Mikhail Andrenkov
```

Comment:
Hi @thonic! Thanks for contributing to The Walrus 🚀

Just a small comment: we write the contributors in a single line ordered alphabetically by last name:
```suggestion
Mikhail Andrenkov, Antonín Hoskovec
```